prisma/migrations/20240709170029_add_rating_to_user_movies/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - Added the required column `rating` to the `UserMovies` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "UserMovies" ADD COLUMN     "rating" DOUBLE PRECISION NOT NULL;
```

Comment:
Not blocking, I believe when developing with prisma locally you can use `prisma db push`, and only create one migration file with `prisma migrate dev`. Not 100% familiar with Prisma, but it would be good to reduce the number of files.